apps/shipments/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .shipment import Device, Shipment, Location, LoadShipment, TransitState, ShipmentState, ExceptionType, \
```

Comment:
I think it'll be easier to maintain this if we just import all models using a wildcard, like `from .shipment import *`, etc.